scheduler/task_priority_cmp.go

```diff
@@ -28,7 +28,7 @@ type byPriority struct{}
 
 func (c *byPriority) name() string { return "task priority" }
 func (c *byPriority) compare(t1, t2 task.Task, _ *CmpBasedTaskComparator) (int, string, error) {
-	reason := "priority is higher"
+	reason := "higher is first"
```

Comment:
"higher priority"?

Reply:
I left this one as is

Reply:
As far as I can tell, we only get the reason string in "taskMoreImportantThan", so couldn't it be opaque that this is by priority despite the name of the struct?